commit c53e54623f537e96a66811e16d48a2523f4833f8
Author: Oliver Kurth <okurth@vmware.com>
Date:   Wed Mar 4 12:07:11 2020 -0800

    Rectify a log spew in vmsvc logging (vmware-vmsvc-root.log)
    
    When a LSI Logic Parallel SCSI controller sits in PCI bus 0
    (SCSI controller 0), the Linux disk device enumeration does not provide
    a "label" file with the controller name.  This results in messages like
    
    'GuestInfoGetDiskDevice: Missing disk device name; VMDK mapping
    unavailable for "/var/log", fsName: "/dev/sda2"'
    
    repeatedly appearing in the vmsvc logging.  The patch converts what
    previously was a warning message to a debug message and thus avoids
    the log spew.
    
    This patch has been made available for open-vm-tools 11.0.x on the
    open-vm-tools github repo.
    
    https://github.com/vmware/open-vm-tools/issues/404

diff --git a/open-vm-tools/services/plugins/guestInfo/diskInfo.c b/open-vm-tools/services/plugins/guestInfo/diskInfo.c
index 878276ce..032c0928 100644
--- a/open-vm-tools/services/plugins/guestInfo/diskInfo.c
+++ b/open-vm-tools/services/plugins/guestInfo/diskInfo.c
@@ -846,9 +846,9 @@ GuestInfoGetDiskDevice(const char *fsName,
     */
    for (indx = 0; indx < partEntry->diskDevCnt; indx++) {
       if (partEntry->diskDevNames[indx][0] == '\0') {
-         g_warning("%s: Missing disk device name; VMDK mapping unavailable "
-                   "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
-                   partEntry->name, fsName);
+         g_debug("%s: Missing disk device name; VMDK mapping unavailable "
+                 "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
+                 partEntry->name, fsName);
          partEntry->diskDevCnt = 0;
          free(partEntry->diskDevNames);
          partEntry->diskDevNames = NULL;
